pydocstringformatter/formatting.py

```diff
@@ -1,39 +1,74 @@
+import abc
 import re
 import tokenize
 
+# pylint: disable=too-few-public-methods
```

Comment:
Could you add this to `pyproject.toml`? I keep ignoring this in my projects anyway 😄

Reply:
Done (We should make this one optional asap, it's very annoying to remove it first thing in all new projects 😄)